spec/user_coffee_shop_query_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../user_coffee_shop_query'
+require_relative '../coffee_shop'
+require_relative '../user'
+
+RSpec.describe UserCoffeeShopQuery do
+  let (:filename) { 'coffee_shops.csv' }
+  let (:coffee_shops) { CoffeeShop.new(filename).all }
+  let (:user) { User.new(12, 43) }
+  let(:coffee_shops) { [{name: "Starbucks Seattle", x: "47.5809", y: "-122.3160"},
+                       {name: "Starbucks SF", x: "37.5209", y: "-122.3340"},
+                       {name: "Starbucks Moscow", x: "55.752047", y: "37.595242"},
+                       {name: "Starbucks Seattle2", x: "47.5869", y: "-122.3368"}] }
+
+  describe "#calculate_distances?" do
```

Comment:
This method is not used anywhere in the project other than in its internals. Shouldn't be better to test the public `#show_closest`?

Reply:
It makes much more sense this way. Fixed it. :+1: